static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -84,11 +83,14 @@ const removeAdSlots = () => {
 };
 
 const setPublisherProvidedId = () => {
-    const user = getUserFromCookie();
-    if (user) {
-        const hashedId = sha1.hash(user.id);
-        window.googletag.pubads().setPublisherProvidedId(hashedId);
-    }
+	// Also known as PPID
+	getUserFromApi((user) => {
```

Comment:
The api is already called once per page view (which is very high traffic)

Does this call it more than once? If so we need to store the response in memory and reuse it.